demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,145 @@
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
+* _adapters_ - implement a common interface to allow Model API wrappers usage with different executors: OpenVINO, ONNX, etc. See [Model API adapters](#model-api-adapters) section
+* _models_ - implement wrappers for each architecture. See [Model API Wrappers](#model-api-wrappers) section
+* _pipelines_ - implement pipelines for model inference and manage the synchronous/asynchronous execution. See [Model API Pipelines](#model-api-pipelines) section
+
+## Building Python* Model API package
+For installation Python (version 3.6 or higher) is required. The installation is available from source.
+
+Use the following command to install Python* Model API from source:
+```sh
+pip install <omz_dir>/demos/common/python
+```
+
+Alternatively, you can generate the package using a wheel. Follow the steps below:
+1. Build the wheel.
+
+```sh
+python <omz_dir>/demos/common/python/setup.py bdist_wheel
+```
+The wheel should appear in the dist folder.
+Name example: `openmodelzoo_modelapi-0.0.0-py3-none-any.whl`
+
+2. Install the package in the clean environment with `--force-reinstall` key.
+```sh
+pip install openmodelzoo_modelapi-0.0.0-py3-none-any.whl --force-reinstall
+```
+
+To verify the package is installed, you might use the following command:
+```sh
+python -c "from openvino.model_zoo import model_api"
+```
+
+> **NOTE**: On Linux and macOS, you may need to type `python3` instead of `python`. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/).
+> For example, on Ubuntu execute the following command to get pip installed: `sudo apt install python3-pip`.
+
+## Model API Wrappers
+
+The Python* Model API package suggests ready-to-use model wrappers, which implement standardized preprocessing/postprocessing functions per "task type" and might be reused in applications as "black-box" models.
+Also, the simple wrapper interface allows the creation of custom wrappers covering different architectures.
+
+The following tasks can be solved with wrappers usage:
+
+| Task type                  | Model API wrappers |
+|----------------------------|--------------------|
+| Background Matting         | `VideoBackgroundMatting`, `ImageMattingWithBackground` |
+| Classification             | `Classification` |
+| Deblurring                 | `Deblurring` |
+| Human Pose Estimation      | `HpeAssociativeEmbedding`, `OpenPose` |
+| Instance Segmentation      | `MaskRCNNModel`, `YolactModel` |
+| Monocular Depth Estimation | `MonoDepthModel` |
+| Named Entity Recognition   | `BertNamedEntityRecognition` |
+|  Object Detection          | `CenterNet`, `DETR`, `CTPN`, `FaceBoxes`, `RetinaFace`, `RetinaFacePyTorch`, `SSD`, `UltraLightweightFaceDetection`, `YOLO`, `YoloV3ONNX`, `YoloV4`, `YOLOF`, `YOLOX` |
```

Comment:
can bullet points will be used inside the table for multiple wrappers line? it will be more accurate, then just comma separation, when you have a lot of wrappers like for object detection

Reply:
Added bullet points using html lists